projects/packages/jitm/src/class-post-connection-jitm.php

```diff
@@ -204,6 +204,74 @@ public static function jitm_jetpack_boost_activate() {
 		);
 	}
 
+	/**
+	 * A special filter used in the CTA of a JITM offering to install the Jetpack Protect plugin.
+	 *
+	 * @return string The new CTA
+	 */
+	public static function jitm_jetpack_protect_install() {
+		return wp_nonce_url(
+			add_query_arg(
+				array(
+					'jetpack-protect-action' => 'install',
+				),
+				admin_url( 'admin.php?page=jetpack' )
+			),
+			'jetpack-protect-install'
+		);
+	}
+
+	/**
+	 * A special filter used in the CTA of a JITM offering to activate the Jetpack Protect plugin.
+	 *
+	 * @return string The new CTA
+	 */
+	public static function jitm_jetpack_protect_activate() {
+		return wp_nonce_url(
+			add_query_arg(
+				array(
+					'jetpack-protect-action' => 'activate',
+				),
+				admin_url( 'admin.php?page=jetpack' )
+			),
+			'jetpack-protect-install'
+		);
+	}
+
+	/**
+	 * A special filter used in the CTA of a JITM offering to install the Jetpack Protect plugin.
```

Comment:
```suggestion
	 * A special filter used in the CTA of a JITM offering to install the Jetpack VideoPress plugin.
```

This comment and the one below should be updated to say this instead

Reply:
Good catch!